doc/operation-and-maintenance/MongooseIM-metrics.md

```diff
@@ -170,111 +258,231 @@ Metrics specific to an extension, e.g. Message Archive Management, are described
+    | `system_info_Metric` | gauge | Erlang system statistics from [`erlang:system_info/1`](https://www.erlang.org/doc/apps/erts/erlang.html#system_info/1). `Metric` can be `port_count`, `port_limit`, `process_count`, `process_limit`, `ets_count` or `ets_limit`. |
+
+=== "Exometer"
 
-| Metric name | Type | Description |
-| ----------- | ---- | ----------- |
-| `[global, system_memory, Metric]` | gauge | Erlang memory statistics from [`erlang:memory/0`](https://www.erlang.org/doc/apps/erts/erlang.html#memory/0). `Metric` specifies the memory type, e.g. `total`, `processes_used`, `atom_used`, `binary`, `ets` or `system`. |
-| `[global, system_info, Metric]` | gauge | Erlang system statistics from [`erlang:system_info/1`](https://www.erlang.org/doc/apps/erts/erlang.html#system_info/1). `Metric` can be `port_count`, `port_limit`, `process_count`, `process_limit`, `ets_count` or `ets_limit`. |
+    | Metric name | Type | Description |
+    | ----------- | ---- | ----------- |
+    | `[global, system_memory, Metric]` | gauge | Erlang memory statistics from [`erlang:memory/0`](https://www.erlang.org/doc/apps/erts/erlang.html#memory/0). `Metric` specifies the memory type, e.g. `total`, `processes_used`, `atom_used`, `binary`, `ets` or `system`. |
+    | `[global, system_info, Metric]` | gauge | Erlang system statistics from [`erlang:system_info/1`](https://www.erlang.org/doc/apps/erts/erlang.html#system_info/1). `Metric` can be `port_count`, `port_limit`, `process_count`, `process_limit`, `ets_count` or `ets_limit`. |
 
 ## Backend metrics
 
 Some extension modules expose histograms with timings of calls made to their backends.
 Please check the documentation of modules that are enabled in your config file, in order to learn if they provide them.
 
-All module backend metrics names use the following convention: `[global, backends, Module, BackendAction]` and `[global, backends, Module, BackendAction, count]`.
-The former is a histogram of operation times. However, the time is not recorded if a backend operation exits with an exception.
-The latter is a number of calls (spiral metric), incremented for *every* call (even a failed one).
-
-Besides these, following authentication metrics are always available:
-
-* `[HostType, backends, auth, authorize]`
-* `[HostType, backends, auth, check_password]`
-* `[HostType, backends, auth, try_register]`
-* `[HostType, backends, auth, does_user_exist]`
+Prometheus metrics have a `host_type` label associated with these metrics, as well as a `function` label describing the backend action.
+Since Exometer doesn't support labels, the host types and backend actions are part of the metric names.
+
+=== "Prometheus"
+
+    | Metric name | Type | Description |
+    | ----------- | ---- | ----------- |
+    | `BackendModule_count` | counter | Number of calls (spiral metric), incremented for *every* call (even a failed one). |
+    | `BackendModule_time` | histogram | Successful operation times. |
```

Comment:
This is just my opinion, but I think the description would be a bit clearer if phrased this way
```suggestion
    | `BackendModule_time` | histogram | Times of successful operations. |
```